Sources/Sentry/SentryProfiler.mm

```diff
@@ -490,39 +504,10 @@ - (void)start
     [self startMetricProfiler];
 }
 
-- (void)stop
-{
-    if (_profiler == nullptr) {
-        SENTRY_LOG_WARN(@"No profiler instance found.");
-        return;
-    }
-    if (!_profiler->isSampling()) {
-        SENTRY_LOG_WARN(@"Profiler is not currently sampling.");
-        return;
-    }
-
-    _profiler->stopSampling();
-    [_metricProfiler stop];
-    SENTRY_LOG_DEBUG(@"Stopped profiler %@.", self);
-}
-
-+ (SentryEnvelopeItem *)envelopeItemForProfileData:(NSDictionary<NSString *, id> *)profile
-                                         profileID:(SentryId *)profileID
-{
-    const auto JSONData = [SentrySerialization dataWithJSONObject:profile];
-    if (JSONData == nil) {
-        SENTRY_LOG_DEBUG(@"Failed to encode profile to JSON.");
-        return nil;
-    }
-
-    const auto header = [[SentryEnvelopeItemHeader alloc] initWithType:SentryEnvelopeItemTypeProfile
-                                                                length:JSONData.length];
-    return [[SentryEnvelopeItem alloc] initWithHeader:header data:JSONData];
-}
```

Comment:
`+[envelopeItemForProfileData:profileID:]` was inlined at the sole callsite on new lines 379-387